0-ad-history-mod-package/simulation/templates/units/han/support_trader.xml

```diff
@@ -8,6 +8,15 @@
     <Civ>han</Civ>
     <SpecificName>Shāngrén</SpecificName>
     <Icon>units/han/support_trader.png</Icon>
+    <History>
+        In the Han dynasty's early days, the people on its northern borders were repeatedly being raided by the Xiognu. The Xiognu were nomads who lived and fought on their horses. Because calvary had never before been used as a primary force in ancient China, the Han were unprepared to fight back. And so, they saw no choice but to pay tribute to the Xiognu in order to  try to make peace. However, despite the tribute payments, the Xiognu kept on pillaging.
```

Comment:
```suggestion
        In the Han dynasty's early days, the people on its northern borders were repeatedly being raided by the Xiognu. The Xiognu were nomads who lived and fought on their horses. Because cavalry had never before been used as a primary force in ancient China, the Han were unprepared to fight back. And so, they saw no choice but to pay tribute to the Xiognu in order to  try to make peace. However, despite the tribute payments, the Xiognu kept on pillaging.
```